packages/knip/fixtures/plugins/prairielearn/question/info.json

```diff
@@ -0,0 +1,19 @@
+{
+    "controller": "pl-rich-text-editor.py",
```

Comment:
PrairieLearn is a framework for creating custom autograded questions. Instructors use this custom format to specify the scripts that should be loaded for their question. Consequently, we have to be extra careful removing dependencies that any instructors are relying on.